.github/workflows/build_test-PR-templates.yml

```diff
@@ -27,11 +26,22 @@ jobs:
         repository: jbosstools/jbosstools-build-ci
         path: build-ci
 
-    - name: Setup Java ${{ matrix.java }}
-      uses: actions/setup-java@v1
+    - name: Setup Java 8
+      uses: actions/setup-java@v2.0.0
       with:
-        java-version: ${{ matrix.java }}
-        java-package: jdk # (jre, jdk, or jdk+fx) - defaults to jdk
+        distribution: 'adopt'
+        java-version: 8
+        java-package: jdk
+        architecture: x64
+    - name: Set JDK_8_LOCATION
+      run: |
+        echo "JDK_8_LOCATION=$JAVA_HOME" >> $GITHUB_ENV
```

Comment:
don't u need to `export` that variable?

Reply:
I do not think so: https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable

Reply:
ok, good to know